pkg/volume/rbd/rbd.go

```diff
@@ -291,7 +291,7 @@ func (r *rbdVolumeProvisioner) Provision() (*v1.PersistentVolume, error) {
 	adminSecretNamespace := rbdDefaultAdminSecretNamespace
 	secretName := ""
 	secret := ""
-	imageFormat := rbdImageFormat1
+	imageFormat := rbdImageFormat2
```

Comment:
Does this require any new packages or configuration to change?  Could this break existing users on upgrade?

Reply:
Negative -- RBD v2 images have been supported for 5+ years and krbd/librbd can transparently open v1 and v2 images. The only thing changing here is that newly created images, by default, will use the v2 on-disk format.